target/linux/ramips/dts/DWR-921-C.dts

```diff
@@ -0,0 +1,132 @@
+/dts-v1/;
+
+#include "mt7620n.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "d-link,dwr-921", "ralink,mt7620n-soc";
+	model = "D-Link DWR-921 C1";
+
+        gpio-keys-polled {
+                compatible = "gpio-keys-polled";
+                #address-cells = <1>;
+                #size-cells = <0>;
+                poll-interval = <20>;
+
+                wps {
+                        label = "wps";
+                        gpios = <&gpio0 2 GPIO_ACTIVE_LOW>;
+                        linux,code = <KEY_WPS_BUTTON>;
+                };
+                reset {
+                        label = "reset";
+                        gpios = <&gpio0 1 GPIO_ACTIVE_LOW>;
+                        linux,code = <KEY_RESTART>;
+                };
+
+        };
+
+        gpio-leds {
+                compatible = "gpio-leds";
+
+                sms {
+                        label = "dwr-921-c1:green:sms";
+                        gpios = <&gpio1 14 GPIO_ACTIVE_LOW>;
+                };
+                lan {
+                        label = "dwr-921-c1:green:lan";
+                        gpios = <&gpio1 15 GPIO_ACTIVE_HIGH>;
+                };
+                sstrengthg {
+                        label = "dwr-921-c1:green:sigstrength";
+                        gpios = <&gpio2 0 GPIO_ACTIVE_LOW>;
+                };
+                sstrengthr {
+                        label = "dwr-921-c1:red:sigstrength";
+                        gpios = <&gpio2 1 GPIO_ACTIVE_LOW>;
+                };
+                4g {
+                        label = "dwr-921-c1:green:4g";
+                        gpios = <&gpio2 2 GPIO_ACTIVE_LOW>;
+                };
+                3g {
+                        label = "dwr-921-c1:green:3g";
+                        gpios = <&gpio2 3 GPIO_ACTIVE_LOW>;
+                };
+
+        };
+
+};
+
+&gpio1 {
+	status = "okay";
+};
+
+&gpio2 {
+	status = "okay";
+};
+
+&spi0 {
+	status = "okay";
+
+	m25p80@0 {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <10000000>;
+
+		uboot: partition@0 {
+			label = "uboot";
+			reg = <0x0 0x40000>;
+			read-only;
+		};
+		
+		uboot_env: partition@40000 {
+			label = "uboot-env";
+			reg = <0x40000 0x10000>;
+			read-only;
+		};
+
+		firmware: partition@50000 {
+			label = "firmware";
+			reg = <0x50000 0xfa0000>;
+		};
+
+		config: partition@ff0000 {
+			label = "config";
+			reg = <0xff0000 0x10000>;
+			read-only;
+		};
+
+	};
+};
+
+&ehci {
+	status = "okay";
+};
+
+&ohci {
+	status = "okay";
+};
+
+&ethernet {
+	mtd-mac-address = <&config 0xe2ac>;
+	mediatek,portmap = "llllw";
+};
+
+&wmac {
+	ralink,mtd-eeprom = <&config 0xe08e>;
+};
+
+&pinctrl {
+	state_default: pinctrl0 {
+		default {
+			ralink,group = "i2c", "wdt", "ephy", "spi refclk", "rgmii1", "rgmii2", "nd_sd";
```

Comment:
- wdt (`<&gpio0 17>`) isn't used as GPIO in your dts
- rgmii2 (`<&gpio2 17>` to `<&gpio2 31>`) isn't used as GPIO in your dts
- nd_sd (`<&gpio2 5>` to `<&gpio2 19>`) isn't used as GPIO in your dts